pyds9/setup_package.py

```diff
@@ -138,6 +139,12 @@ def post_build_ext_hook(cmd):
         print(" ".join(compile_cmd))
         sp.check_call(compile_cmd)
 
+    # run make clean
+    xpa_dir = [i for i in libxpa.include_dirs if 'xpa' in i][0]
+
+    with cd(xpa_dir):
+        sp.check_call(['make', 'distclean'])
```

Comment:
@montefra xpa's distclean now removes xpa.pc and con.h ... thanks for pointing this issue out.